pkg/dbutils/postgresql_utils.go

```diff
@@ -42,6 +42,14 @@ type postgresDB interface {
 	Close() error
 }
 
+// PostgresAssetManagerIface represents the methods of the PG asset manager
+type PostgresAssetManagerIface interface {
+	Init() error
+	Close() error
+	QueryOne(target interface{}, query string, args ...interface{}) error
+	QueryAllCharts(query string, args ...interface{}) ([]*models.Chart, error)
+}
+
```

Comment:
It's not obvious to me why you're creating a separate interface here, when you already have the `AssetManager` interface (which has `Init() error` and `Close() error`). Do we not want `QueryOne` and `QueryAllCharts` available on the mongo asset manager as well?

I see it's really just something you need for testing here, so perhaps it doesn't make sense to go implementing those for mongodb, hence creating the specific one here? (If that's the case, perhaps update the comment to save people having to figure that out).

Also, you can probably do something like:
```
type PostgresAssetManagerIface interface {
    AssetManager
    QueryOne(....
    ...
}
```
so it's more explicit that this just adds the two methods to the existing iface?

Reply:
> I see it's really just something you need for testing here, so perhaps it doesn't make sense to go implementing those for mongodb, hence creating the specific one here? (If that's the case, perhaps update the comment to save people having to figure that out).

Indeed, that's the case

> so it's more explicit that this just adds the two methods to the existing iface?

:+1: